### ISSUE TITLE
What hardware is supported?

### ISSUE THREAD

**Issue by mithro** [SpinalHDL/SaxonSoc#56]
It seems most boards are listed under a `deprecated` directory @ https://github.com/SpinalHDL/SaxonSoc/tree/dev-0.1/hardware/deprecated ?

**Comment by Dolu1990** [comment]
Yes, basicaly, to move forward with the flow and the limited man power, i decided to port ArtyA7 and Ulx3s linux SoC and focus on it in the dev-0.1.
I moved others in deprecated, until somebody is interrested into bringing some back.

**Comment by Dolu1990** [comment]
So the current status is :
dev-0.1 => ArtyA7 + Ulx3s  running linux 5.0.9 SMP
dev-0.2 => ArtyA7 running linux 5.10.3, Ulx3s will be ported as soon it is stabilized
